finesse/hardware/__init__.py

```diff
@@ -18,16 +18,14 @@
 else:
     from .temperature.dp9800 import DP9800  # type: ignore
 
-dp9800: DP9800
 opus: OPUSInterface
 
 
 def _init_hardware():
     global opus, dp9800
 
-    opus = OPUSInterface()
-
     dp9800 = DP9800()
```

Comment:
This will raise an error unless you pass the `--dummy-dp9800` argument, because it expects a `Serial` as an argument. (`mypy` would ordinarily catch this, but it's missing it because of the `type: ignore` shenanigans above.) I'd just drop the `if "--dummy-dp9800"` business above and unconditionally use the `DummyTemperatureMonitor` for now. We'll obvs replace this bit anyway once when we tackle #141.